lib/dor/services/client.rb

```diff
@@ -118,6 +118,10 @@ def create_release_tag(object:, release:, what:, to:, who:)
         # @raise [UnexpectedResponse] when the response is not successful.
         # @return [boolean] true on success
         delegate :publish, to: :objects
+
+        # Notify goobi system of a new object
+        delegate :notify_goobi, to: :objects
```

Comment:
Looks good! Can you also add a corresponding test hereish? https://github.com/sul-dlss/dor-services-client/blob/master/spec/dor/services/client_spec.rb#L15-L75